.github/workflows/busted.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit", "luajit-openresty" ]
+        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit" ]
```

```suggestion
        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit" ] # "luajit-openresty"
```

Lets comment this out rather than nuke it as a reminder to try again someday. There is *some* utility in running it because it does sometimes give different results. That fork has diverged a bit from other LuaJITs and when it isn't crashing it is good to know it works there too.